ksoap2-j2se/src/main/java/org/ksoap2/transport/OkHttpServiceConnectionSE.java

```diff
@@ -1,5 +1,7 @@
 package org.ksoap2.transport;
 
+import okhttp3.OkHttpClient;
```

Comment:
please leave it with single line imports .. otherwise this looks good to me

Reply:
Ahh my bad on this one. I used the "optimize imports" from my IDE and it neglected single line imports. Thanks for updating it though. :+1: